examples/fluids/qfunctions/stg_shur14.h

```diff
@@ -297,12 +297,39 @@ CEED_QFUNCTION(ICsSTG)(void *ctx, CeedInt Q, const CeedScalar *const *in, CeedSc
   const CeedScalar       nu     = stg_ctx->newtonian_ctx.mu / rho;
 
   CeedPragmaSIMD for (CeedInt i = 0; i < Q; i++) {
-    const CeedScalar x_i[3]     = {x[0][i], x[1][i], x[2][i]};
-    const CeedScalar dXdx[3][3] = {
-        {q_data[1][i], q_data[2][i], q_data[3][i]},
-        {q_data[4][i], q_data[5][i], q_data[6][i]},
-        {q_data[7][i], q_data[8][i], q_data[9][i]}
-    };
+    const CeedScalar x_i[3] = {x[0][i], x[1][i], x[2][i]};
+    CeedScalar       dXdx[3][3];
+    {
+      const CeedScalar J11  = J[0][0][i];
+      const CeedScalar J21  = J[0][1][i];
+      const CeedScalar J31  = J[0][2][i];
+      const CeedScalar J12  = J[1][0][i];
```

Comment:
I think long term it's probably better to have this as a QFunction helper and call it through there, but I'm ok with this for now.